src/tui/state.rs

```diff
@@ -451,6 +471,7 @@ impl<'a> State<'a> {
             Tab::StaticAnalysis => vec![
                 ("Enter", "Details"),
                 ("/", "Search"),
+                ("s", "Human Readable"),
```

Comment:
Why do we use the `s` key here? We can go with `r` (`r`eadability) and change the description to "readability" or something similar. The latter is not necessary - my point is "s" key is not that intuitive :)

Also, I just realized pressing `r` on any tab re-runs the executable, we should probably fix that to happen on dynamic tab analysis only. We should probably fix that 🫠

Reply:
r is already used in keycode::Char for TraceCalls.  It can't reach both patterns with the way the keycodes are designed.